src/styles/details.css

```diff
@@ -0,0 +1,50 @@
+.county-details {
```

Comment:
# Design
- **NB: Allow me to sum up everything here since the styling is also concerned**. 
- Please follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including:
- Typography: font face, size, and weight
- Layout: composition and space between elements.
- Kindly select one main color for the app (it does **not** need to be the same color as in the design guidelines)
- Also create a "Back" button. when the user clicks on the button (<), it navigates to the home page
- Original Design:
![1](https://user-images.githubusercontent.com/63972670/203406894-dc97e572-aa59-47d8-8fda-039adf966452.PNG)
- **YOUR DESIGN**:
- 
![Capture](https://user-images.githubusercontent.com/63972670/203407079-ef8aa4fb-69e1-4550-b5ac-0d7d9cda367d.PNG)
-
![Capture1](https://user-images.githubusercontent.com/63972670/203407147-f67033d9-0208-4f24-8ab4-a3e530f03441.PNG)
-

![as](https://user-images.githubusercontent.com/63972670/203407200-199c427c-3fc4-4ffe-bf01-bf30f865521d.PNG)

Reply:
Done